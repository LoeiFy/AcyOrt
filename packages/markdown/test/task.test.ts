import Markdown from '../src'

it('task', () => {
  const m = new Markdown()
  const t0 = '- [x] this is a complete item'
  const t1 = '* Item 1'

  expect(m.render(t0)).toMatchSnapshot()
  expect(m.render(t1)).toMatchSnapshot()
})
