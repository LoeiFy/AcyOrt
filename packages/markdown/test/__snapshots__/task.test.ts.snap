// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`task 1`] = `
"<ul>
<li style="list-style: none"><input checked="" disabled="" type="checkbox"> this is a complete item</li>
</ul>
"
`;

exports[`task 2`] = `
"<ul>
<li>Item 1</li>
</ul>
"
`;
